force-app/main/default/classes/ContactTriggerHandler.cls

```
@@ -0,0 +1,60 @@
+public class ContactTriggerHandler {
+    public void handleBeforeInsert(List<Contact> newContacts) {
+        for (Contact con : newContacts) {
+            if (String.isBlank(con.DummyJSON_Id__c)) {
+                Integer randomInt = Math.round(Math.random() * 100);
+                con.DummyJSON_Id__c = String.valueOf(randomInt);
+            }
+        }
+    }
+
+    public void handleAfterInsert(List<Contact> newContacts) {
+        for (Contact con : newContacts) {
+            if (!String.isBlank(con.DummyJSON_Id__c)) {
+                try {
+                    Integer dummyId = Integer.valueOf(con.DummyJSON_Id__c);
+                    if (dummyId <= 100) {
+                        DummyJSONCallout.getDummyJSONUserFromId(con.Id);
+                        
+                    }
+                } catch (Exception e) {
+                    System.debug('Error parsing DummyJSON_Id__c in after insert: ' + e.getMessage());
```

Watch your PMD Warnings and Errors.

Also, don't use Try/Catch if you are just going to debug the error. This can lead to silent failures that you need to fix in your code. Try/Catch is meant to recover gracefully from errors and do something when the error happens.